src/Aliases.jl

```diff
@@ -76,7 +76,6 @@
 
 # Allow backwards compatibility after removal of Oscar.Graphs module.
 Graphs = Oscar
```

Comment:
This is of course not your mistake at all, but since i am seeing this:
```suggestion
const Graphs = Oscar
```
CC @lkastner